examples/lessons/14/app.js

```diff
@@ -0,0 +1,291 @@
+import {
+  GL, AnimationLoop, Model, Geometry, loadTextures, loadFiles, Vector3, setParameters, Program
+} from 'luma.gl';
+
+import {Matrix4, radians} from 'math.gl';
+
+const FRAGMENT_LIGHTING_VERTEX_SHADER = `\
+precision highp float;
+
+attribute vec3 positions;
+attribute vec3 normals;
+attribute vec2 texCoords;
+
+uniform mat4 uMMatrix;
+uniform mat4 uVMatrix;
+uniform mat4 uPMatrix;
+
+varying vec2 vTextureCoord;
+varying vec4 vTransformedNormal;
+varying vec4 vPosition;
+
+void main(void) {
+  vPosition = uMMatrix * vec4(positions, 1.0);
+  gl_Position = uPMatrix * uVMatrix * vPosition;
+  vTextureCoord = texCoords;
+  vTransformedNormal = uMMatrix * vec4(normals, 0.0);
+}
+`;
+
+const FRAGMENT_LIGHTING_FRAGMENT_SHADER = `\
+#ifdef GL_ES
+precision highp float;
+#endif
+
+varying vec2 vTextureCoord;
+varying vec4 vTransformedNormal;
+varying vec4 vPosition;
+
+uniform float uMaterialShininess;
+uniform bool uShowSpecularHighlights;
+uniform bool uUseLighting;
+uniform bool uUseTextures;
+uniform vec3 uAmbientColor;
+uniform vec3 uPointLightingLocation;
+uniform vec3 uPointLightingSpecularColor;
+uniform vec3 uPointLightingDiffuseColor;
+uniform sampler2D uSampler;
+
+void main(void) {
+    vec3 lightWeighting;
+    if (!uUseLighting) {
+        lightWeighting = vec3(1.0, 1.0, 1.0);
```

Comment:
Nit: rename `lightWeighting` to `lightWeight`  and same for other variables?

Reply:
It is the same as the examples in original webgl-lessons. Maybe keep it as this way?